business-central-parent/business-central-distribution-wars/src/main/shared/swarm/sample-swarm-config.yml

```diff
@@ -0,0 +1,70 @@
+# Sample Swarm config for KIE Drools Workbench
+swarm:
+  management:
+    blocking:
+      timeout: 900 # This might need to be higher if the jar has timeout issues on startup
+    security-realms:
+      ApplicationRealm: # almost the same as default config in WildFly 10.1.0.Final
+        local-authentication:
+          default-user: local
+          allowed-users: local
+          skip-group-loading: true
+        properties-authentication:
+          path: application-users.properties
+          plain-text: true
+        properties-authorization:
+          path: application-roles.properties
+# Optional logging
+  logging:
+    console-handlers:
+      CONSOLE:
+        level: INFO
+        named-formatter: COLOR_PATTERN
+    pattern-formatters:
+      PATTERN:
+        pattern: "%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
+      COLOR_PATTERN:
+        pattern: "%K{level}%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"
+    periodic-rotating-file-handlers:
+      FILE:
+        file:
+          path: server.log
+        suffix: .yyyy-MM-dd
+        named-formatter: PATTERN
+    root-logger:
+      handlers:
+      - CONSOLE
+      - FILE
+  security:
+    security-domains:
+      other:
+        classic-authentication:
+          login-modules:
+            kieLoginModule:
+              code: org.kie.security.jaas.KieLoginModule
+              flag: optional
+              module: deployment.business-central-webapp.war
+datasource:
+  management:
+    wildfly:
+      admin: admin
+
+# This configuration is required to make the users system work.
+org:
+  uberfire:
+    ext:
+      security:
+        management:
+          wildfly:
+            cli:
+              user: admin
+  kie:
+    server: # Sample connection to KIE Server.
+      id: kieserver1
+      user: kieserver
+      pwd: kieserver1!
+      location: http://localhost:8230/kie-server/services/rest/server
+      controller:
+        user: kieserver
+        pwd: kieserver1!
+      controller: http://localhost:8080/business-central/rest/controller
```

Comment:
@domhanak There is a small concern about if this works. Ideally the controller branch should not break, but if I have the url in the higher "controller:" I get an error on startup.

Reply:
the `controller` tags here do not make any difference and the correct url would be `http://localhost:8080rest/controller`

Reply:
the `controller` tags here do not make any difference and the correct url would be `http://localhost:8080rest/controller`